EXTENSIONS.md

```diff
@@ -117,9 +117,28 @@ instead of the output format described above.  E.g.
 ```
 
 In particular, the `linkerd check` command will invoke the check command for
-each extension installed in the cluster and will request json output.  To
-preserve forwards compatibility, it is recommended that the check command should
-ignore any unknown flags.
+each extension installed in the cluster and will request json output.
+`linkerd check` may optinally invoke your extension if not installed in the
+cluster (see `linkerd-name config` below to opt-in).  To preserve forwards
+compatibility, it is recommended that the check command should ignore any
+unknown flags.
+
+### `linkerd-name config`
+
+This subcommand is optional, and enables an extension to opt-in to being
+executed as part of `linkerd check`, even when there is no corresponding
+extension on the cluster. To opt-in, the output of `linkerd-name config` should
```

Comment:
Uh sorry one last concern. Do we really want to call this _config_. Are we never going to want to allow any linkerd components to have a `config` subcommand (as kubectl does)? Can we more clearly mark this as internal with like `linkerd-name _extension-config`?

Also, this should never be exposed to users via help.